sklearn/metrics/_plot/roc_curve.py

```diff
@@ -20,7 +20,7 @@ class RocCurveDisplay(_BinaryClassifierCurveDisplayMixin):
     a :class:`~sklearn.metrics.RocCurveDisplay`. All parameters are
     stored as attributes.
 
-    Read more in the :ref:`User Guide <visualizations>`.
+    Read more in the :ref:`User Guide <roc_metrics>`.
```

Comment:
For `ROCCurveDisplay`, the `visualizations` link does show how to use the visualization api with `ROCCurveDisplay`: https://scikit-learn.org/stable/visualizations.html#visualizations

This kind of depends on what a reader wants to learn. "What is ROC Curve" or "How to use RocCurveDisplay"

Reply:
We've resorted to linking both e.g.,

https://github.com/scikit-learn/scikit-learn/blob/7131d9488dfb8edd6ae042caca57dd76523f395b/sklearn/metrics/_plot/regression.py#L25-L28